mock/middle_ware.go

```diff
@@ -0,0 +1,27 @@
+package mock
+
+import (
+	"net/http"
+
+	"github.com/influxdata/influxdb"
+	platcontext "github.com/influxdata/influxdb/context"
+)
+
+// NewMiddlewareHandler create a mocked middleware handler.
+func NewMiddlewareHandler(handler http.Handler, auth influxdb.Authorizer) http.Handler {
```

Comment:
Seems like this is a very generic name, since it specifically is am authentication middleware mock.

Reply:
sure